recipe/patches/0004-use-C-17-everywhere.patch

```diff
@@ -0,0 +1,24 @@
+From 025938fdbfa0ed950afedd9d773e1f6b499950bd Mon Sep 17 00:00:00 2001
+From: "H. Vetinari" <h.vetinari@gmx.com>
+Date: Thu, 18 May 2023 17:14:37 +1100
+Subject: [PATCH 4/5] use C++17 everywhere
+
+---
```

Comment:
Can you add a link to the original patch please?

Reply:
put link in meta.yaml

Reply:
I would patch it in the patch directly.